must-gather/collection-scripts/gather_cluster_logging_operator_resources

```diff
@@ -38,16 +38,12 @@ if [ $NAMESPACE == "openshift-logging" ]; then
   done
 fi
 
-log "Gathering data for 'clusterlogging' and 'clusterlogforwarder' from namespace: $NAMESPACE"
-for r in "clusterlogging" "clusterlogforwarder" ; do
-  names="$(oc -n $NAMESPACE get $r --ignore-not-found -o jsonpath='{.items[*].metadata.name}' | sort -u)"
-  for name in ${names[@]}; do
-    data="$(oc -n $NAMESPACE get $r ${name} --ignore-not-found -o yaml)"
-    if [ "$data" != "" ] ; then
-      echo $data > "${clo_folder}/${r}_${name}.yaml"
-    fi
-  done
-done
+log "Gathering data for 'clusterlogging' and 'clusterlogforwarder' from ${BASE_COLLECTION_PATH}/namespaces/openshift-logging/logging.openshift.io/"
```

Comment:
This is incorrect as it breaks must-gather for the multi CLF scenario.  We should be getting these from ANY namespace that has them.  @cahartma didn't you already address this change?

Reply:
I did and something is strange here.  Still trying to figure out what I'm looking at.  How is all that logic missing?

Reply:
We are passing the namespace into that script...    so it's collecting from all namespaces that contain a cl or clf
https://github.com/openshift/cluster-logging-operator/blob/master/must-gather/collection-scripts/gather#L29-L43